CommonTools/TrackerMap/src/TrackerMap.cc

```diff
@@ -3623,8 +3623,7 @@ saveAsSingleLayer=false;
 TLegend* TrackerMap::buildLegend() {
   if ( legInfos_.empty() ) {
     legInfos_.reserve(8);
-    Double_t x[4] = {.2,.7,.6,.2};
-    Double_t y[4] = {.5,.1,.9,.5};
+    double x[4],y[4];
     legInfos_.push_back(new TPolyLine(4,x,y));
```

Comment:
this is actually worse
https://github.com/root-project/root/blob/v6-06-00-patches/graf2d/graf/src/TPolyLine.cxx#L93
x and y are uninitialized and could even lead to segfaults in the constructor.

Is there a reason the ```new TPolyLine(0)``` was considered unacceptable?

Reply:
Hi @slava77, I misunderstood the previous comment. I was trying to match the way how people did in the same script. I will change to the empty TPolyLine now.